recipe/meta.yaml

```diff
@@ -22,30 +22,33 @@ source:
   sha256: 252819173f4d3737c516c5b81aa3c82a92a21f666633705ecf239a0b33e95899
 
 build:
-  number: 2
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  number: 3
+  script:
+    # Ensure we can find the vips package
+    - pkg-config --exists --print-errors vips
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ stdlib('c') }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    # 2021/12/30 hmaarrfk
-    # I'm really not sure how to ensure that things get cross compiled correctly
-    # Therefore, we rely on ABI mode for cross compiled architectures
-    # - cffi                                   # [build_platform != target_platform]
-    # - pkgconfig                              # [build_platform != target_platform]
-    # - libvips                                # [build_platform != target_platform]
+    - pkg-config                          # [not win]
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - cffi >=1.0                          # [build_platform != target_platform]
+    - pkgconfig >=1.5                     # [build_platform != target_platform]
+    - libvips                             # [build_platform != target_platform]
   host:
     - python
     - pip
     - setuptools >=61
     - cffi >=1.0
-    - pkgconfig >=1.5   # [not win]
+    - pkgconfig >=1.5                     # [not win]
     - libvips
     - glib
+    - expat
+    - xorg-xorgproto                      # [linux]
```

Comment:
Without this, `pkg-config --exists --print-errors vips` fails with:
```
Package xproto was not found in the pkg-config search path.
Perhaps you should add the directory containing `xproto.pc'
to the PKG_CONFIG_PATH environment variable
Package 'xproto', required by 'x11', not found
```

See: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1327429&view=logs&j=5b44946f-b0b5-570f-e735-950feabb4722&t=d53332be-ec07-565d-9b53-75513afd6eda&l=1128